cmake/FindCOCAL.cmake

```diff
@@ -0,0 +1,53 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Find the COCAL initial data code (assumed repository or documentation link)
+#
+# Pass `COCAL_ROOT` to the CMake build configuration to set up the following
+# targets:
+#
+# - COCAL::Exporter: Functionality to load COCAL volume data and interpolate to
+#   arbitrary points.
+
+if(NOT DEFINED COCAL_ROOT OR COCAL_ROOT STREQUAL "")
+  set(COCAL_ROOT $ENV{COCAL_ROOT})
+endif()
+
+if (DEFINED COCAL_ROOT AND NOT COCAL_ROOT STREQUAL "")
+  
+  # Find the COCAL library
+  find_library(
+    COCAL_LIB
+    NAMES libcocal.a
```

Comment:
this should not contain the `lib` or the `.a` (at least the latter is system dependent).